pkg/controller/inspection.go

```diff
@@ -55,7 +55,12 @@ func (c *Controller) inspectPod() error {
 				if !isLspExist {
 					delete(pod.Annotations, fmt.Sprintf(util.AllocatedAnnotationTemplate, podNet.ProviderName))
 					delete(pod.Annotations, fmt.Sprintf(util.RoutedAnnotationTemplate, podNet.ProviderName))
-					if _, err := c.config.KubeClient.CoreV1().Pods(pod.Namespace).Patch(context.Background(), pod.Name, types.JSONPatchType, generatePatchPayload(pod.Annotations, "replace"), metav1.PatchOptions{}, ""); err != nil {
+					patch, err := util.GenerateStrategicMergePatchPayload(oriPod, pod)
```

Comment:
The inspection will delete annotations and, will mergePatch bring the deleted annotations back?

Reply:
No, see test case 4